app/views/about/index.html.erb

```diff
@@ -60,7 +60,8 @@
             <% end %>
             </tbody>
           </table>
-          <%= _('To enable a provider, either install the OS package (e.g. foreman-libvirt) or enable the bundler group for development setup (e.g. libvirt).') %>
+          <%= _('You can enable additional compute resources. For more information, see') %>
+          <%= link_to _('provisioning hosts'), external_link_path(type: 'docs', section: "Provisioning_Hosts", chapter: ""), :rel => "external" %>.
```

Comment:
Please use the helpers introduced in https://github.com/theforeman/foreman/commit/b566ea8ab3c56f79edb0ed9a1caa3c28038af961. I think this should work:
```suggestion
          <%= link_to _('provisioning hosts'), documentation_url('Provisioning_Hosts', type: 'docs'), :rel => "external" %>.
```